### PR TITLE
feat: localization for chat bot micro app

### PR DESCRIPTION
**Pull request by Abhishek01039**

### CODE REVIEW

File: lib/src/localization/localization_delegate.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import 'package:etiya_chatbot_flutter/src/localization/generated/app_localizations.dart';
+import 'package:flutter/material.dart';
+
+class ToggPluginAppLocalizationsDelegate
```
Comment (EnesKaraosman) [inline lib/src/localization/localization_delegate.dart:4]: This **package** is not a plugin, so ToggMicroAppXXX might be more appropriate? @Abhishek01039

File: lib/src/localization/i10n.dart [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        return 'en-US';
+      case tr:
+        return 'tr-TR';
+      default:
+        return '';
+    }
+  }
+
+  /// Get the flag by language code
+  static String getFlag(String code) {
```
Comment (EnesKaraosman) [inline lib/src/localization/i10n.dart:41]: We don't need flags currently..

File: lib/src/localization/generated/app_localizations_tr.dart [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  String get email_address => 'E-posta Adresi';
+
+  @override
+  String get enter_non_empty_password => 'Boş olmayan şifreyi girin';
+
+  @override
+  String get password => 'Şifre';
+
+  @override
+  String get login => 'Giriş yapmak';
```
Comment (EnesKaraosman) [inline lib/src/localization/generated/app_localizations_tr.dart:26]: pls change it to 'Giriş'

File: lib/src/localization/generated/app_localizations_tr.dart [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  String get submit => 'Gönder';
+
+  @override
+  String get enter_valid_email => 'Geçerli e-posta girin';
+
+  @override
+  String get email_address => 'E-posta Adresi';
+
+  @override
+  String get enter_non_empty_password => 'Boş olmayan şifreyi girin';
```
Comment (EnesKaraosman) [inline lib/src/localization/generated/app_localizations_tr.dart:20]: 'Lütfen şifre alanını boş bırakmayınız'

File: lib/src/localization/generated/app_localizations_tr.dart [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  AppLocalizationsTr([String locale = 'tr']) : super(locale);
+
+  @override
+  String get submit => 'Gönder';
+
+  @override
+  String get enter_valid_email => 'Geçerli e-posta girin';
+
+  @override
+  String get email_address => 'E-posta Adresi';
```
Comment (EnesKaraosman) [inline lib/src/localization/generated/app_localizations_tr.dart:17]: 'Email'

File: lib/src/localization/generated/app_localizations_tr.dart [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+/// The translations for Turkish (`tr`).
+class AppLocalizationsTr extends AppLocalizations {
+  AppLocalizationsTr([String locale = 'tr']) : super(locale);
+
+  @override
+  String get submit => 'Gönder';
+
+  @override
+  String get enter_valid_email => 'Geçerli e-posta girin';
```
Comment (EnesKaraosman) [inline lib/src/localization/generated/app_localizations_tr.dart:14]: 'Geçerli bir email giriniz'

File: l10n.yaml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+arb-dir: lib/src/localization
```
Comment (EnesKaraosman) [inline l10n.yaml:1]: does localization package automatically generates under `localization/generated` folder, or you need to specifically define here?

**Review by EnesKaraosman — CHANGES_REQUESTED** [review]

File: l10n.yaml (hunk 7 above)
Comment (Abhishek01039) [reply l10n.yaml:1]: Need to drag and drop the generated files to lib/localization/generated files

File: lib/src/localization/localization_delegate.dart (hunk 1 above)
Comment (Abhishek01039) [reply lib/src/localization/localization_delegate.dart:4]: `ChatBotAppLocalizationsDelegate` how about this @EnesKaraosman ?

File: README.md [hunk 8]
```diff
@@ -2,20 +2,32 @@
...
+- MacOS
+- Windows
 
 ### How to integrate?
 
-See [the readme of example project](example/README.md)
\ No newline at end of file
+See [the readme of example project](example/README.md)
+
+### Localisation
```
Comment (EnesKaraosman) [inline README.md:23]: Typo: `Localization`

**Pull request merged**